src/battle/area_flo2/5CE690.c

```diff
@@ -18,9 +18,19 @@ INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218620_5CECB0);
 
 INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_80218774_5CEE04);
 
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188A8_5CEF38);
-
-INCLUDE_ASM(s32, "battle/area_flo2/5CE690", func_802188EC_5CEF7C);
+ApiStatus func_802188A8_5CEF38(void) {
```

Comment:
The signature of all script-api functions (any returning an `ApiStatus`) should be
```c
ApiStatus func_802188A8_5CEF38(ScriptInstance* script, s32 isInitialCall);
```
regardless of if it uses the args or not (in this case, neither are used).

Reply:
Ditto for `func_802188EC_5CEF7C`.